core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/worker/neo/PauseUtil.scala

```diff
@@ -0,0 +1,74 @@
+package edu.uci.ics.amber.engine.architecture.worker.neo
+
+import java.util.concurrent.CompletableFuture
+
+object PauseUtil{
```

Comment:
This file is not just a `util`. It contains the actual pause state. This is kind of the truth that has to be contacted for all pause related works. Think of a better name. How about `PauseControlModule`?